modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -0,0 +1,280 @@
+[%collapsible]
+====
+* `Microsoft.Authorization/roleAssignments/delete`
+====
+
+.Required permissions for deleting compute resources
+[%collapsible]
+====
+* `Microsoft.Compute/disks/delete`
+* `Microsoft.Compute/galleries/delete`
+* `Microsoft.Compute/galleries/images/delete`
+* `Microsoft.Compute/galleries/images/versions/delete`
+* `Microsoft.Compute/virtualMachines/delete`
+====
+
+.Required permissions for deleting identity management resources
+[%collapsible]
+====
+* `Microsoft.ManagedIdentity/userAssignedIdentities/delete`
+====
+
+.Required permissions for deleting network resources
+[%collapsible]
+====
+* `Microsoft.Network/dnszones/read`
+* `Microsoft.Network/dnsZones/A/read`
+* `Microsoft.Network/dnsZones/A/delete`
+* `Microsoft.Network/dnsZones/CNAME/read`
+* `Microsoft.Network/dnsZones/CNAME/delete`
+* `Microsoft.Network/loadBalancers/delete`
+* `Microsoft.Network/networkInterfaces/delete`
+* `Microsoft.Network/networkSecurityGroups/delete`
+* `Microsoft.Network/privateDnsZones/read`
+* `Microsoft.Network/privateDnsZones/A/read`
+* `Microsoft.Network/privateDnsZones/delete`
+* `Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete`
+* `Microsoft.Network/publicIPAddresses/delete`
+* `Microsoft.Network/virtualNetworks/delete`
+====
+[NOTE]
+====
+You must remove the following permissions while deleting the private {product-title} cluster on Azure.
+
+* `Microsoft.Network/dnszones/read`
+* `Microsoft.Network/dnsZones/A/read`
+* `Microsoft.Network/dnsZones/A/delete`
+* `Microsoft.Network/dnsZones/CNAME/read`
+* `Microsoft.Network/dnsZones/CNAME/delete`
+====
+
+.Required permissions for checking health of resources
+[%collapsible]
+====
+* `Microsoft.Resourcehealth/healthevent/Activated/action`
+* `Microsoft.Resourcehealth/healthevent/Resolved/action`
+* `Microsoft.Resourcehealth/healthevent/Updated/action`
+====
+
+.Required permissions for deleting resource group
+[%collapsible]
+====
+* `Microsoft.Resources/subscriptions/resourcegroups/delete`
+====
+
+.Required permissions for deleting storage resources
+[%collapsible]
+====
+* `Microsoft.Storage/storageAccounts/delete`
+* `Microsoft.Storage/storageAccounts/listKeys/action`
+====
+
+[NOTE]
+====
+You can create a custom role with all the permissions and assign that custom role to the service principal. To create a custom role in the Azure portal, see link:https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles[Azure custom roles] in the Azure documentation.
+====
+
+[NOTE]
+====
+For installing the {product-title} cluster, you must scope the permissions to subscription level. Later, you can re-scope these permissions to the installer created resource group level. If the public DNS zone is present in a different resource group, then the network DNS zone related permissions must always be at subscription level.
+By default, the {product-title} installer assigns the Azure identity the `Contributor` role.
```

Comment:
```suggestion
By default, the {product-title} installation program assigns the Azure service principal the `Contributor` role.
```
@jinyunma is my terminology correct here? I'm looking for a more specific term than "identity".

Reply:
@bscott-rh I think Azure identity is the correct term here, and it should not be service principal. Please check [here](https://docs.openshift.com/container-platform/4.12/installing/installing_azure/installing-azure-user-infra.html#installation-azure-create-resource-group-and-identity_installing-azure-user-infra).